packages/e2e-test-utils/src/index.js

```diff
@@ -52,6 +52,14 @@ export {
 } from './observe-focus-loss';
 export { openDocumentSettingsSidebar } from './open-document-settings-sidebar';
 export { openPublishPanel } from './open-publish-panel';
+export {
```

Comment:
What's the motivation to make those utils part of the public API? Can they be located in the subfolder of `packages/e2e-tests`?

Reply:
🤔 No particular reasons, to be honest.
I'm not familiar with Guten's e2e, and just figured that utils are supposed to go in `e2e-test-utils`. 😅 

As far as I can see, these utils are only relevant for the performance tests, so my first thought was to create a utils folder into `packages/e2e-tests/specs/performance`, but then I couldn't find any other "local utils".

Reply:
We can start with local `utils` in the top level next to `specs` 😄 

You can also try inside `specs/performance` unless Jest picks them up as tests.